crates/ethereum-rpc/src/lib.rs

```diff
@@ -220,12 +219,12 @@ fn register_rpc_methods<C: sov_modules_api::Context, Da: DaService>(
         info!("eth module: eth_feeHistory");
         let mut params = params.sequence();
 
-        let block_count: U64HexOrNumber = params.next()?;
+        let block_count: Index = params.next()?;
```

Comment:
Was U64HexOrNumber replaced with Index type in reth?

this should parse both 30 and 0x1e

Reply:
https://reth.rs/docs/reth_rpc_types/struct.Index.html

Yes but it's usize now, not u64

Reply:
checked it its fine